test/ex_audit_test.exs

```diff
@@ -245,4 +249,110 @@ defmodule ExAuditTest do
 
     assert 2 = Repo.aggregate(query, :count, :id)
   end
+
+  describe "insert_all/3" do
+    setup %{opts: opts} do
+      return =
+        Repo.insert_all(
+          User,
+          [
+            [
+              name: "foo",
+              email: "foo@foo.com",
+              inserted_at: ~N[2021-01-01 01:57:57],
+              updated_at: ~N[2021-01-01 01:57:57]
+            ],
+            [
+              name: "bar",
+              email: "bar@bar.com",
+              inserted_at: ~N[2021-01-01 01:57:58],
+              updated_at: ~N[2021-01-01 01:57:58]
+            ]
+          ],
+          opts
+        )
+
+      %{return: return}
+    end
+
+    @describetag opts: []
```

Comment:
`@describetag` is the same as `@tag` on each `test` within the current `describe` block. Unsure what you wanted to achieve with this.

Reply:
agreed, could have added this as a `@tag` in the `test` below, gonna change it

Reply:
No, I meant that I wanted to know why you are setting the `opts` tag at all. Since you're not using it anywhere, the only use I could think of that you could use it in a filter

Reply:
it is used in all tests https://github.com/ZennerIoT/ex_audit/pull/63/files#diff-7d13cb3ba44758097343775d3a3eabb44fedaf4f8e4f481ac3d2dcf7b23e90f1R272

Reply:
Yeah sorry. I didn't know that this will make the opts available in the setup functions. So your usage of this is fine.